doc/configuring.rst

```diff
@@ -48,7 +48,7 @@ Both `name` and `command`, however, can be undefined. A build step without a nam
 a build step without a command will do nothing, but will still appear in log (and have a step number).
 
 For storing such parameters :mod:`universum.configuration_support` provides a class :class:`Step`, that has
-:ref:`a list of various step parameters <keys>`. Build steps can be :ref:`added, multiplied <combining>` and
+a list of various step parameters. Build steps can be :ref:`added, multiplied <combining>` and
```

Comment:
I understand that `Step` itself is a link to key list, but please leave a more obvious "blue" link too for those not attentive enough.